mods/voidmulti.sc2mod/base.sc2data/gamedata/modeldata.xml

```diff
@@ -352,6 +352,34 @@
             <Payload value="Repair_Bot_Beam_01"/>
         </Events>
     </CModel>
+    <CModel id="Observer">
+        <Radius value="0.438000"/>
+        <ScaleMax value="1.000000,1.000000,1.000000"/>
+        <ScaleMin value="1.000000,1.000000,1.000000"/>
+        <SelectionRadius value="0.515000"/>
+        <ShadowRadius value="0.515000"/>
+    </CModel>
```

Comment:
Observer's model size increased by 17.5%

- Model radius increased from 0.4 to 0.43 (+ 7,5%)
  - shouldn't it be ~ 0,5053?
  - I am not sure what this property does for units, though :) I believe there was something with when the model is visible or hidden in fog, so models do not pop in
- ScaleMin and ScaleMax increased from 0. 85 to 1.0 (+17.5%)
- SelectionRadius and ShadowRadius (for blob shadows) increased from 0.4702 to 0.5150 (+ 9.5%)
  - is that visually fine since it was not scaled with the same percentage?
- I would have expected SelectionOffset to be changed as well since the model grew in all dimensions